exporters/logging/src/main/java/io/opentelemetry/exporter/logging/LoggingSpanExporter.java

```diff
@@ -15,12 +15,23 @@
 
 /** A Span Exporter that logs every span at INFO level using java.util.logging. */
 public class LoggingSpanExporter implements SpanExporter {
-  private static final Logger logger = Logger.getLogger(LoggingSpanExporter.class.getName());
+  // Visible for testing
+  static final Logger logger = Logger.getLogger(LoggingSpanExporter.class.getName());
 
   @Override
   public CompletableResultCode export(Collection<SpanData> spans) {
+    // We always have 32 + 16 + name + several whitespace, 60 seems like an OK initial guess.
+    StringBuilder sb = new StringBuilder(60);
     for (SpanData span : spans) {
-      logger.log(Level.INFO, "span: {0}", span);
+      sb.setLength(0);
+      sb.append(span.getName())
+          .append(" ")
+          .append(span.getTraceId())
```

Comment:
any thoughts to prefixing `"t"` before the trace id and `"s"` before the span id?

Reply:
The drastically different length of the values makes me ok without the prefix.

Reply:
yeah, I guess at the moment, we can't get shorter traceIds into here, since we'll always zero-pad if we get on. 👍